backend/webapi/Features/Parties/LicenceDeclaration.cs

```diff
@@ -98,7 +98,7 @@ public class CommandHandler(IPlrClient plrClient, PidpDbContext context, IClock
             }
             catch (Exception ex)
             {
-                this.context.BusinessEvents.Add(CollegeIDEnteredInCollegeLicneceSearchErrorCase.Create(party.Id, command.CollegeCode.Value.ToString(), this.clock.GetCurrentInstant()));
+                this.context.BusinessEvents.Add(CollegeLicenceSearchError.Create(party.Id, command.CollegeCode.Value.ToString(), this.clock.GetCurrentInstant()));
```

Comment:
This is logging when we don't find the college licence, however we want to log when there is an error in the search operation. So we need to move the logging from here